### PR TITLE
Attempt to fix authors

### LINKED ISSUE #158: Feed Author Problem
There is a problem in the feed author section.

https://blog.getbootstrap.com/feed.xml

```
<author>
<name>{"twitter"=>"getbootstrap"}</name>
</author>
```

Ekran Alıntısı

### PR DESCRIPTION
**Pull request by XhmikosR**
Fixes #158 

Preview: <https://deploy-preview-162--bootstrapblog.netlify.app/>
Feed: <https://deploy-preview-162--bootstrapblog.netlify.app/feed.xml>

### CODE REVIEW

**Comment by XhmikosR** [comment]
@mdo which solution of the two do you prefer?

File: _config.yml [hunk 1]
```diff
@@ -32,8 +32,10 @@ themes:          "https://themes.getbootstrap.com/"
 opencollective:  "https://opencollective.com/bootstrap"
 
 # Social
-# Default posts' author
-author:         "mdo"
+# Default author info
+author:
+  name:         "Mark Otto, Jacob Thornton, and Bootstrap contributors"
```
Comment (mdo) [inline _config.yml:37]: Nearly every blog post has been written by me, save a few. Maybe we stick to me as the default author and the `getbootstrap` handle? We could then use that in the signoff of every post that I do.

Comment (XhmikosR) [reply _config.yml]: I will do it properly when I have some time. :)

**Comment by XhmikosR** [comment]
All right, let's go with this solution I guess. If we need to add a new post author, we specify them in the front matter and add a new entry in _data/authors.yml.

**Pull request merged**